platform/commonUI/general/src/directives/MCTSplitPane.js

```diff
@@ -135,8 +139,8 @@ define(
                     splitter.css(anchor.opposite, "auto");
 
                     last.css(anchor.edge, firstSize + splitterSize + 'px');
-                    last.css(anchor.opposite, "0px");
-                    position = firstSize + splitterSize;
```

Comment:
I notice that a lot of adjustments for splitter size have been removed. I'm unclear on why that would be necessary here